recipes-debian/cluster-glue/cluster-glue_debian.bb

```diff
@@ -0,0 +1,176 @@
+#base recipe: meta-cgl/meta-cgl-common/recipes-cgl/cluster-glue/cluster-glue_1.0.12.bb
+#base branch: dunfell
+#base commit: b8529657086ba4d308ee8cc99b720aaedaa3ae18
+
+DESCRIPTION = "Cluster Glue is a set of libraries, tools and utilities suitable for \
+the Heartbeat/Pacemaker cluster stack. In essence, Glue is everything that \
+is not the cluster messaging layer (Heartbeat), nor the cluster resource manager \
+(Pacemaker), nor a Resource Agent."
+HOMEPAGE = "http://clusterlabs.org/"
+LICENSE = "GPLv2 & LGPLv2.1"
```

Comment:
There is two files under the BSD -3-clause but I'm not sure these files are used or not.

```
Files: replace/daemon.c
       replace/alphasort.c
Copyright: 1998-2015 The Regents of the University of California
License: BSD-3-clause
```

https://metadata.ftp-master.debian.org/changelogs//main/c/cluster-glue/cluster-glue_1.0.12-12_copyright

Reply:
I see. Thank you for checking it.